src/SonarQube.Client/Services/SonarQubeClient.cs

```diff
@@ -63,45 +65,66 @@ public SonarQubeClient(HttpMessageHandler messageHandler, TimeSpan requestTimeou
                         request.OrganizationKey, request.Page, request.PageSize); // TODO: should handle optional params
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)